doc/vimspectorpy.txt

```diff
@@ -203,7 +203,23 @@ You could choose 'rxvt' or force 'xterm' using:
 
 >
   let g:vimspectorpy#launcher = "rxvt"
+
 <
+                          g:vimspectorpy#tmux#size
+
+Set the size of the split opened, when using the tmux launcher. Default is
+equivalent to
+
+  let g:vimspectorpy#tmux#size = 10
+
+
+                          g:vimspectorpy#tmux#split
+
+Set the orientation of the split opened, when using the tmux launcher. Valid
+options are "v" for vertical and "h" for horizontal. Default is equivalent to
+
+  let g:vimspectorpy#tmux#split = "v"
```

Comment:
To quote a block of ex-commands verbatim, place a greater than (>) character
at the end of the line before the block and a less than (<) character as the
first non-blank on a line following the block.